### PR TITLE
Remove Rockets patch on Brayns after version 1.1

### PR DESCRIPTION
**Pull request by NadirRoGue**

### CODE REVIEW

**Comment by NadirRoGue** [comment]
Just to be sure that what I did is what I expect: This will make any variant (including develop and latest from master branch) after 1.1 not have such patch applied, is that correct?

File: var/spack/repos/builtin/packages/brayns/package.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def patch(self):
             )
         for cmake_filename in find(self.stage.source_path, "CMakeLists.txt"):
             filter_file(r'\$\{GLEW_LIBRARIES\}', 'GLEW', cmake_filename)
-        if self.spec.satisfies('@1.0:'):
+        if self.spec.satisfies('@1.0:1.1'):
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns/package.py:70]: If you want 1.1 still to have the patch applied, then, yes, this is correct.

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/brayns/package.py (hunk 1 above)
Comment (NadirRoGue) [reply var/spack/repos/builtin/packages/brayns/package.py:70]: Yes, 1.1 needs it but any newer release or the current stuff in the master branch doesn't need it. Thanks a lot!

Comment (matz-e) [reply var/spack/repos/builtin/packages/brayns/package.py:70]: Feel free to merge, all good from my side. (also, I think this should capture releases like `1.1.1`, IIRC, so I hope your next one will be `1.2`)

Comment (NadirRoGue) [reply var/spack/repos/builtin/packages/brayns/package.py:70]: Yes, next one is 1.2, thanks.

**Pull request merged**